File: fabric/src/main/java/net/impleri/playerskills/fabric/PlayerSkillsFabric.java
package net.impleri.playerskills.fabric;

import net.fabricmc.api.ModInitializer;
import net.impleri.playerskills.PlayerSkills;

public class PlayerSkillsFabric implements ModInitializer {
    public void onInitialize() {
        PlayerSkills.init();
    }
}

File: fabric/src/main/scala/net/impleri/playerskills/fabric/PlayerSkillsFabric.scala
package net.impleri.playerskills.fabric

import net.fabricmc.api.ModInitializer
import net.impleri.playerskills.PlayerSkills

case class PlayerSkillsFabric() extends ModInitializer {
  private val INTEGRATIONS: FabricIntegrationLoader = FabricIntegrationLoader(PlayerSkills.STATE)

  override def onInitialize(): Unit = {
    PlayerSkills.init()
  }
}
